### PR TITLE
[ALLUXIO-1809]Add renameStore operation in KeyValueSystem

### PR DESCRIPTION
**Pull request by TeddyBear1314**
https://alluxio.atlassian.net/browse/ALLUXIO-1809

### CODE REVIEW

File: keyvalue/server/src/main/java/alluxio/master/keyvalue/KeyValueMaster.java [hunk 1]
```diff
@@ -314,6 +317,43 @@ void checkIsCompletePartition(long fileId, AlluxioURI uri) throws InvalidPathExc
   }
 
   /**
+   *Renames one completed key-value store.
```
Comment (apc999) [inline keyvalue/server/src/main/java/alluxio/master/keyvalue/KeyValueMaster.java]: NIT: a space before "Renames" please

File: core/server/src/proto/journal/key_value.proto [hunk 2]
```diff
@@ -28,6 +28,12 @@ message DeleteStoreEntry {
 }
 
 // next available id: 3
+message RenameStoreEntry {
+ optional int64 old_store_id = 1;
```
Comment (apc999) [inline core/server/src/proto/journal/key_value.proto]: NIT: indent 2 spaces?

**Comment by apc999** [comment]
@TeddyBear1314 Thanks for the patch! I left two very minor comments. Otherwise the PR looks good to me!

**Comment by TeddyBear1314** [comment]
Thanks for the review!
I have changed the code.
I should have checked my code more carefully.
Anyway, I'll do better!

**Comment by apc999** [comment]
+1 LGTM

**Pull request merged**